README.md

```diff
@@ -162,6 +162,13 @@ In order to avoid reference integrity issues when deleting acquisition units tha
 * When client sends `GET /acquisitions-units/units?query=<cql>` and `<cql>` does not contain criteria by `isDeleted`, the logic will search for records with `isDeleted==false`.
 * To get all the units regardless of `isDeleted` value, the request should be like `GET /acquisitions-units/units?query=isDeleted=* AND (<cql>)`
 
+### Kafka Integration
+during tenant init, mod-orders creates producer with numPartitions which can be customized with env vars below:
+* DI_ORDER_CREATED_READY_FOR_POST_PROCESSING_PARTITIONS
+* DI_ORDER_CREATED_PARTITIONS
+* DI_PENDING_ORDER_CREATED_PARTITIONS
+Default value is 1
```

Comment:
do you know why we need to create kafka topics during module enable? it there was some issue with that maybe we need to include our acquisition topics as well?

Reply:
hi @SerhiiNosko! we've implemented the creation of DI topics programmatically in mod-srm, srs, inventory, data-import, the reason was to reduce time of import files as during import topics were being created on broker side via auto create, also we want to have more control over topics. if we'll turn of auto create feature on kafka broker, then acquisitions topics should be created as well